### PR TITLE
Alert for membership lapsed 

### PR DESCRIPTION
**Pull request by weedySeaDragon**
### PT Story: User gets notified if memberships lapsed
https://www.pivotaltracker.com/story/show/161583992

### Changes proposed in this pull request:
1.  Locale: Added placeholders for  text
2. Updated locale text (and updated the email image below)  ~Created draft emails~
3. Created Alert with spec and feature
4. Added to `load_conditions` with a **draft** schedule

### Screenshots (Optional):
email-membership_lapsed-sv-html

Ready for review:
@thesuss  @patmbolger

### CODE REVIEW

**Review by patmbolger — APPROVED** [review]

File: config/locales/en.yml [hunk 1]
```diff
@@ -1042,6 +1042,15 @@ en:
           how_to_pay_fee: You can pay your H-branding for another 12 months by paying the fee. This makes you visible on the map.
           company_link_login_msg: " (You may need to login first before you can see the button to pay)."
 
+      membership_lapsed:
+        subject: "Your SHF membership has expired. Don't give up the benefits of membership!"
+        greeting: *email_greeting
+        message_text:
+          expire_alert_html: "Your SHF membership term ended on %{expire_date}.  (list benefits.  ...company is no longer visible to the public.)"
```
Comment (thesuss) [inline config/locales/en.yml]: remove "(list benefits.  ...company is no longer visible to the public.)"

File: config/locales/sv.yml [hunk 2]
```diff
@@ -1045,6 +1045,14 @@ sv:
           how_to_pay_fee: Betala den ett år till via länken nedan. Detta gör att du fortsätter synas på kartan och fortsätter ha ett H-märkt företag.
           company_link_login_msg: " (du kan behöva logga in först innan du ser knappen för att betala)."
 
+      membership_lapsed:
+        subject: "Your SHF membership has expired. Don't give up the benefits of membership!"
```
Comment (thesuss) [inline config/locales/sv.yml]: Ditt medlemskap har löpt ut

File: config/locales/sv.yml [hunk 3]
```diff
@@ -1045,6 +1045,14 @@ sv:
...
+        greeting: *email_greeting
+        message_text:
+          expire_alert_html: "Your SHF membership term ended on %{expire_date}.  (list benefits.  ...company is no longer visible to the public.)"
```
Comment (thesuss) [inline config/locales/sv.yml]: Ditt medlemsskap gick ut %{expire_date}.

File: config/locales/sv.yml [hunk 4]
```diff
@@ -1045,6 +1045,14 @@ sv:
...
+          renew_membership: You can renew your membership and keep taking advantage of SHF member benefits by logging in and paying your membership fee.
```
Comment (thesuss) [inline config/locales/sv.yml]: Du kan förnya ditt medlemsskap och fortsätta dra nytta av medlemsfördelarna genom att logga in och betala.

File: config/locales/sv.yml [hunk 5]
```diff
@@ -1045,6 +1045,14 @@ sv:
...
+          account_login_html: "%{account_link} (you may need to login first before the link works)."
```
Comment (thesuss) [inline config/locales/sv.yml]: Du betalar på ditt konto: %{account_link} (du behöver vara inloggad för att länken ska fungera).

**Comment by weedySeaDragon** [comment]
@thesuss  - I updated the text.  Proofread to make sure it's ok, please.  (I updated the screenshot in the PR description)

**Comment by thesuss** [comment]
the name will be the real name, right?

**Comment by weedySeaDragon** [comment]
Yes, the real name will be displayed. :-)  (This is a user created with a factory method and so the default first name is "Firstname" and the default last name is "Lastname")

Totally worth asking, just in case. 👍

**Pull request merged**